docs/docs/advanced-use/release-pipelines.md

```diff
@@ -0,0 +1,176 @@
+
+1. Go to your project in the Flagsmith dashboard.
+2. Navigate to the **Release Pipelines** section.
+3. Click **Create Release Pipeline**.
+
+### Step 2: Configure Pipeline Settings
+
+![On Enter](/img/release-pipelines/pipeline-configuration.png)
+
+**Pipeline Information:**
+- **Name**: Give your pipeline a descriptive name (e.g., "Web App Release Pipeline").
+
+**Pipeline Configuration:**
+- Add multiple stages to represent your release process.
+- Each stage can represent an environment or phase in your release process.
+
+## Pipeline Stages
+
+### Stage Configuration
+
+Each stage in your pipeline consists of:
+
+1. **Stage Name**: A descriptive name for the stage (e.g., "Development", "Staging", "Production", "Turn on beta users").
+2. **Trigger**: When the stage should be activated.
+3. **Actions**: What should happen when the stage is triggered.
+
+### Stage Triggers
+
+**On Enter** (`ON_ENTER`)
+
+![On Enter](/img/release-pipelines/on-enter.png)
+
+- Triggers immediately when a flag is added to this stage.
+- Use this for immediate actions when flags reach a particular stage.
+
+**Wait For** (`WAIT_FOR`)
+
+![Wait For](/img/release-pipelines/wait-for.png)
+
+- Triggers after waiting for a specified amount of time.
+- Configure the wait time in minutes, hours, or days.
+- Use this for gradual rollouts or testing periods.
+
+### Stage Actions
+
+![Wait For](/img/release-pipelines/actions.png)
+
+**Toggle Feature Actions:**
+- **Enable flag for everyone**: Enables the feature flag for all users in the environment.
+- **Disable flag for everyone**: Disables the feature flag for all users in the environment.
+- **Enable flag for segment**: Enables the feature flag for a specific user segment.
+- **Disable flag for segment**: Disables the feature flag for a specific user segment.
+
+## Publishing a Release Pipeline
+
+### Draft Status
+After a release pipeline is created and before it is published, it remains in draft mode. In this state:
+A release pipeline is inactive and does not process feature flags
+- The configuration remains editable.
+- You can modify stages, triggers, and actions freely.
+- Once published, the pipeline becomes immutable and operational.
+
+### Publishing Process
+
+:::warning
+
+Once published, a release pipeline cannot be modified. Ensure your configuration is correct before publishing.
+
+:::
+
+1. Complete your pipeline configuration.
+2. Review all stages, triggers, and actions.
+3. Click **Publish Pipeline**.
+4. Confirm the action when prompted.
+
+![Publish Release Pipeline](/img/release-pipelines/publish-release-pipeline.png)
+
+
+## Adding Flags to Release Pipelines
+
+:::warning
+
+In alpha phase there are no automatic rollbacks, you'd need to revert changes manually.
+
+:::
+
+### From the Dashboard
+
+1. Navigate to your feature flags.
+2. Select the flag you want to add to a pipeline.
+3. Click the **Add to Release Pipeline** option.
+4. Select the pipeline.
+5. Confirm the selection.
```

Comment:
```suggestion
1. Navigate to your feature flags.
1. Select the flag you want to add to a pipeline.
1. Click the arrow on the "Update Feature Value" button to expand options.
1. Click the **Add to Release Pipeline** option.
1. Select the pipeline.
1. Confirm the selection.
```

Feel free to rephrase.

Side note. We discussed offline about managing flags from within the stage card @ Release Pipeline configuration, which will be a great UX improvement in the future. Now while the steps above are the only way to use the feature, adding a feature flag to an ongoing pipeline should be easier to find IMO. I know this is out of scope at this point but here's raising it for a product-level discussion.